### DISCUSSION TITLE
Accessing `emscriptenGetAudioObject` and other emscripten functions after building with `-s MODULARIZE=1`

### DISCUSSION THREAD

**Discussion by danieljamesross** [emscripten-core/emscripten#21288] [Q&A] [answered]
How can I access the emscripten functions, like `emscriptenGetAudioObject`, from javascript land after building with the `-s MODULARIZE=1` flag?

**Comment by kangtastic** [comment, marked as answer]
Had to do this recently. Pass `-sEXPORTED_RUNTIME_METHODS="emscriptenGetAudioObject"` as per the first note here: https://emscripten.org/docs/api_reference/module.html

**Comment by danieljamesross** [comment]
Ah, thank you. So easy when you know how!
